customerio-reactnative.podspec

```diff
@@ -28,7 +28,9 @@ Pod::Spec.new do |s|
   s.default_subspec = "nopush"
 
   s.subspec 'nopush' do |ss|
```

Comment:
Maybe we need to move away from wording `nopush` because otherwise, it can get confusing. how about `base/common` ?

Reply:
This name would be for internal purposes, only which is why I included docs into this file explaining it. However, I do understand what you mean by `nopush` being confusing. 

I wouldn't mind a word `generic-push` or `base-push` which implies that the RN SDK does require some sort of push setup, but it's not specific to the service yet.

Reply:
Sweet, I am approving it as this isn't a blocker, feel free to update it to a more generic name now or later.